### ISSUE TITLE
Slider to move entire landscape up

### ISSUE THREAD

**Issue by 42willow** [Ju1-js/25th-hour-dynamic#14]
I'll work on this after #9 

It would be nice to have extra space for icons down the bottom and a bit less sky.
![image](https://github.com/Ju1-js/25th-hour-dynamic/assets/70784752/e7ab39f4-f51d-4445-b208-866f31adf903)

**Comment by 42willow** [comment]
I'll work on this.

**Comment by 42willow** [comment]
https://github.com/42Willow/25th-hour-dynamic/tree/landscape-slider

**Comment by Ju1-js** [comment]
Merged!
